develop/devguide/Connector/ImpactOfProtocolChanges/TableParameterChanges/ChangeDisplayKey.md

```diff
@@ -0,0 +1,92 @@
+---
+uid: ChangeDisplayKey
+---
+
+# Change display key
+
+Changing the display key of a table is considered a major change.
+
+## Impact
+
+This has the same impact as changing a parameter description. The following DataMiner features can be impacted if used:
+
+- Filters
+- Automation scripts
+- Visio files
+- Reports
+- Web API
+
+*DIS MCC*
+
+|            | Error message | Description                                                                                                        |
+|------------|---------------|--------------------------------------------------------------------------------------------------------------------|
+| **2.39.4** | FormatChanged | Table display key was changed from {oldSyntax} '{oldFormat}' to {newSyntax} '{newFormat}'. Table PID '{tablePid}'. |
+| **2.39.5** | FormatRemoved | Table display key previously defined via '{oldSyntax}' was removed. Table PID '{tablePid}'.                        |
+
+## Workarounds
+
+### Display key column exists
+
+If a display key already exists, and you want to change its format, the following routine should be implemented to avoid impact by default:
+
+- R/W (standalone) discreet parameter to select the naming format
```

Comment:
This is not clear. Could you clarify?

Reply:
which part exactly is not clear? in any case, I would already suggest following improvements:
1. "If a display key is already defined in the connector, and you want to change....."
2. "Read/Write (standalone) discreet parameter allowing the user to chose the desired display key format"

Is this enough to make things clearer to you Marieke?